scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -378,12 +378,16 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             }
 
             function validateForPublish(item) {
+                var requiredFields = config.requiredMediaMetadata;
                 if (item.type === 'picture') {
-                    if (item.alt_text === null || _.trim(item.alt_text) === '') {
-                        notify.error(gettext('Alt text is mandatory for image publishing! ' +
-                            'Edit crops to fill in the alt text.'));
-                        return false;
-                    }
+                    // required media metadata fields are defined in superdesk.config.js
+                    _.each(requiredFields, function (key) {
+                        if (item[key] == null || _.isEmpty(item[key])) {
+                            notify.error(gettext('Required field ' + key + ' is missing. ' +
```

Comment:
would be good to use `$interpolate` here to avoid having multiple strings to translate:
```
notify.error($interpolate(gettext('Required field {{ key }} is missing. ...'))({key: key}));
```